htmlproofer/plugin.py

```diff
@@ -14,10 +14,25 @@
 import requests
 import urllib3
 
+try:
+    import lxml
```

Comment:
Any reason not to simply make `lxml` a hard dependency?

Reply:
Mostly that it is a heavier dependency so I was being conservative. That said, lxml has wheels available for most platforms so I don't think it's unreasonable to just depend on it

Reply:
Just pushed a commit to do this